### PR TITLE
building main page and login page

### PR DESCRIPTION
**Pull request by YousefQwasmeh**
#3

### CODE REVIEW

File: public/html/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <title>Login and Registration</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="library booking" />
+    <meta name="keywords" content="library_booking" />
+    <link rel="shortcut icon" href="../favicon.ico" />
+    <link rel="stylesheet" type="text/css" href="../css/index.css" />
+  </head>
+  <body>
+      <img id="bgImg" src="">
+    <ul>
```
Comment (eadehemingway) [inline public/html/index.html]: i imagine 'bgImg' means 'background image' but its not very clear, make it 'backgroundImg' ?

File: public/html/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <ul class="left">
+        <li><a href="#books">Home</a></li>
+        <li><a href="#books2">myBooks</a></li>
+      </ul>
+
+      <ul class="right">
+        <li><a class="logout" href="#home">Logout</a></li>
+        <li><a href="#m" class="userName">userName</a></li>
+      </ul>
```
Comment (eadehemingway) [inline public/html/index.html]: what is "#m" ?

File: public/html/login.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                  <input
+                    id="usernamesignup"
+                    name="usernamesignup"
+                    required="required"
+                    type="text"
+                    placeholder="mysuperusername690"
+                  />
+                </p>
+                <p>
+                  <label for="emailsignup" class="youmail" data-icon="e">
```
Comment (eadehemingway) [inline public/html/login.html]: in many many places you are naming things not using camel case, it makes it hard to read. e.g 'emailsignup' should be 'emailSignup'

File: public/html/login.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+                  <input type="submit" value="Login" />
+                </p>
+                <p class="change_link">
+                  <a href="#toregister" class="to_register">Sign up</a>
+                </p>
+              </form>
+            </div>
+
+            <div id="register" class="animate form">
+              <form action="mysuperscript.php" autocomplete="on">
```
Comment (eadehemingway) [inline public/html/login.html]: what is "mysuperscript.php" and why is it the action for your form?

File: public/html/index.html (hunk 1 above)
Comment (YousefQwasmeh) [reply public/html/index.html]: done

File: public/html/index.html (hunk 2 above)
Comment (YousefQwasmeh) [reply public/html/index.html]: done

File: public/html/login.html (hunk 3 above)
Comment (YousefQwasmeh) [reply public/html/login.html]: done

File: public/html/login.html (hunk 4 above)
Comment (YousefQwasmeh) [reply public/html/login.html]: done

**Pull request merged**